extension-manifest-v3/src/utils/trackerdb.js

```diff
@@ -11,19 +11,23 @@
 
 import * as engines from './engines.js';
 
-let promise = engines.init('trackerdb').then(() => {
+let promise = engines.init(engines.TRACKERDB_ENGINE).then(() => {
   promise = null;
 });
 
-export async function getMetadata(request) {
-  if (promise) await promise;
+export function getMetadata(request) {
+  if (promise) {
+    console.warn('TrackerDB not ready yet');
```

Comment:
Good to have a warning on that. Do you expect it to be triggering often? (And if so, what functionality is impacted during that time?)

Reply:
all engines should start as soon as possible. On MV3 the consequence of delayed trackerdb startup would be lack of detailed stats in the panel in MV2 some trackers may not be blocked - but there is nothing we can do about it.